src/components/chat/chat.tsx

```diff
@@ -38,6 +44,27 @@ export const Chat: FC<Props> = ({ roomId, room }) => {
     setMessages((prev) => [...prev, newMessage]);
   };
 
+  const handleMessage = (event: { target: { value: string } }) => {
+    const inputValue = event.target.value;
+
+    validateForm(inputValue);
+
+    if (!messageError && inputValue.length <= 150) {
```

Comment:
Define 150 as a global constant or check if we already have one

Reply:
Tbh now I don't even see the point in separating `validateForm` from this function. It's way more clear if the parsing is done here and in the `else` clause we also set the error state